stable/prometheus-blackbox-exporter/values.yaml

```diff
@@ -42,6 +42,21 @@ config:
         no_follow_redirects: false
         preferred_ip_protocol: "ip4"
 
+extraConfigmapMounts: []
+  # - name: certs-configmap
+  #   mountPath: /etc/secrets/ssl/
+  #   subPath: certificates.crt # (optional)
```

Comment:
Sorry, I missed it:  You don't describe defaultMode.

Reply:
Currently the defaultmode is already optional, since if you give it an empty value (or don't mention it) in the values.yaml, the kubernetes json schema will give it a default value of '420'

Reply:
But still, you don't document it in values.yaml. ;)

Reply:
Added an example in the values.yaml :)